src/Data/Store/TH/Internal.hs

```diff
@@ -262,6 +263,11 @@ deriveTupleStoreInstance n =
 deriveGenericInstance :: Cxt -> Type -> Dec
 deriveGenericInstance cs ty = plainInstanceD cs (AppT (ConT ''Store) ty) []
 
+deriveGenericInstanceFromName :: Name -> Q Dec
+deriveGenericInstanceFromName n = do
+    tvs <- map VarT . dtTvs <$> reifyDataType n
+    return $ deriveGenericInstance (map storePred tvs) (appsT (ConT n) tvs)
```

Comment:
This is necessary when `store` tries to generate an instance for `Store (TyVarBndr flag)`. Before `deriveGenericInstance` was used, and it would fail to notice that `TyVarBndr` has a type parameter.